### PR TITLE
Fixes #6, Updated User Guide

### LINKED ISSUE #5: Update photos, roles and general information in `AboutUs` page

### LINKED ISSUE #6: Update user guide from AddressBook to SociaLite

### PR DESCRIPTION
**Pull request by Zacchua**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -160,33 +289,94 @@ AddressBook data are saved in the hard disk automatically after any command that
 
 ### Editing the data file
 
-AddressBook data are saved as a JSON file `[JAR file location]/data/addressbook.json`. Advanced users are welcome to update data directly by editing that data file.
+AddressBook data are saved as a JSON file `[JAR file location]/data/SociaLite.json`. Advanced users are welcome to update data directly by editing that data file.
```
Comment (stanley-1) [inline docs/UserGuide.md:292]: Replace the term  "AddressBook" with "SociaLite".

File: docs/UserGuide.md [hunk 2]
```diff
@@ -160,33 +289,94 @@ AddressBook data are saved in the hard disk automatically after any command that
...
 
 <div markdown="span" class="alert alert-warning">:exclamation: **Caution:**
 If your changes to the data file makes its format invalid, AddressBook will discard all data and start with an empty data file at the next run.
```
Comment (stanley-1) [inline docs/UserGuide.md:295]: Replace the term "AddressBook" with "SociaLite".

File: docs/UserGuide.md [hunk 3]
```diff
@@ -160,33 +289,94 @@ AddressBook data are saved in the hard disk automatically after any command that
...
+* `list` followed by `forward 2` creates a file to store the contact details of Charlie Decker.
+* `find Annie` followed by `forward 1` creates a file to store Annie Baker’s contact details.
 
 --------------------------------------------------------------------------------------------------------------------
 
-## FAQ
+## FAQ <a name="faq"></a>
 
 **Q**: How do I transfer my data to another Computer?<br>
 **A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous AddressBook home folder.
```
Comment (stanley-1) [inline docs/UserGuide.md:358]: Replace the term "AddressBook" with "SociaLite".

**Review by stanley-1 — COMMENTED** [review]
Thanks for compiling the UG, Zac! 

Looks great overall and there are just some minor formatting adjustments to make as listed below. 

Separately, for future iterations, perhaps we should change all references to the term "AddressBook" (lines 292 & 298) or "address book" (lines 169, 177, etc.) with "SociaLite" or "the App".

File: docs/UserGuide.md (hunk 1 above)
Comment (Zacchua) [reply docs/UserGuide.md:292]: Oh right i missed that out! Thanks!

**Review by david-eom — COMMENTED** [review]
Thanks Zac, LGTM!

**Pull request merged**